api/v1beta1/managedosversion_types.go

```diff
@@ -59,9 +59,7 @@ type ManagedOSVersionSpec struct {
 	UpgradeContainer *upgradev1.ContainerSpec `json:"upgradeContainer,omitempty"`
 }
 
-type ManagedOSVersionStatus struct {
-	fleetv1.BundleStatus `json:""` //nolint
-}
+type ManagedOSVersionStatus struct{}
```

Comment:
I tested on main branch by creating a standalone ManagedOSVersion, and also by using a ManagedOSVersionChannel to create some more.
None of them had any `status`, so this is not in use.

Reply:
@davidcassany already suggested to purge this status :D
I can confirm is not in use, so it's safe.